packages/server/modules/core/utils/formatting.ts

```diff
@@ -0,0 +1,2 @@
+export const toMegabytesWith1DecimalPlace = (bytes: number) =>
```

Comment:
You can just probably move this helper to `@speckle/shared` import it from there, cause its currently used (& duplicated) across multiple packages:
![image](https://github.com/user-attachments/assets/9be56647-8c87-4f70-80e3-ccf61f3e6cc8)